src/core/qgsvectorlayerundopassthroughcommand.h

```diff
@@ -70,6 +65,14 @@ class CORE_EXPORT QgsVectorLayerUndoPassthroughCommand : public QgsVectorLayerUn
      */
     void setError();
 
+    QString mSavePointId;
+
+  protected:
```

Comment:
Protected is already specified above. But I wonder if this should be made private, so the setter can make sure that the error string and the error flag are in sync?